### PR TITLE
Added refresh rate for picture entity

### PR DESCRIPTION
**Pull request by kkellner**
## Proposed change

The hui-image used in picture-entity card has a hard-coded refresh rate of 10 seconds.  To provide more flexibility in the image refresh rate a new user-configurable attribute has been added to the picture-entity card.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/frontend/pull/5466

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by kkellner** [comment]
Code PR:  https://github.com/home-assistant/frontend/pull/5466

File: source/_lovelace/picture-entity.markdown [hunk 1]
```diff
@@ -45,6 +45,10 @@ aspect_ratio:
   required: false
   description: "Forces the height of the image to be a ratio of the width. You may enter a value such as: `16x9`, `16:9`, `1.78`."
   type: string
+refresh_rate:
+  required: false
+  description: "Refresh rate of the image in seconds."
+  type: string
```
Comment (frenck) [inline source/_lovelace/picture-entity.markdown:51]: Please add the `default: ` key.

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @kkellner!

Left a small comment, could you please take a look?

Thanks! 👍

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by frenck** [comment]
Closing this PR as the parent PR has been closed.

**Comment by EvanMulawski** [comment]
Can this be reopened and reconsidered? In my opinion, there is no reason to not have this be customizable when a tenet of HA is to be customizable. If the device and the user's environment can support a live video feed, which is one of the options for this card, there is no reason it couldn't support a faster snapshot refresh rate.

**Pull request closed without merging**